### PR TITLE
Add Dead or Alive: Dimensions

### PR DESCRIPTION
**Pull request by SeyaLeNeko**
Adds Dead or Alive: Dimensions.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/dead-or-alive-dimensions/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Dead or Alive: Dimensions"
+description = "Dead or Alive: Dimensions (DOAD) is a fighting game developed by Team Ninja and released by Tecmo Koei and in some regions by Nintendo for the Nintendo 3DS. This made it the first Dead or Alive title for a Nintendo system, and the series's first fighting game for any handheld."
```
Comment (Leo626) [inline games/dead-or-alive-dimensions/game.dat]: The `for the Nintendo 3DS` part could be omitted.

**Comment by Leo626** [comment]
@SeyaLeNeko Just need to address the description.

**Comment by SeyaLeNeko** [comment]
What's wrong with the description??

**Comment by Leo626** [comment]
This comment https://github.com/citra-emu/citra-games-wiki/pull/384#discussion_r202544085.

**Pull request merged**